### ISSUE TITLE
Broken links on mkdocs sites

### ISSUE THREAD

**Issue by iranzo** [technote-space/broken-link-checker-action#96]
Hi 
I got https://github.com/citellusorg/citellus/issues/665 raised for a broken link.

I think it's related with a file used with mkdocs (read the docs) to generate documentation.

Is there an easy way to 'skip it' so that it doesn't report as failed? (as the final rendering process fixes it?)

Thanks!

**Comment by technote-space** [comment]
@iranzo 
This action uses `broken-link-checker`.
Please try `excludedkeywords` option.
https://github.com/stevenvachon/broken-link-checker#excludedkeywords
https://github.com/technote-space/broken-link-checker-action/blob/master/action.yml#L46-L48

```yaml
    steps:
      - name: Broken Link Check
        uses: technote-space/broken-link-checker-action@v1
        with:
          EXCLUDED_KEYWORDS: *magui.md
```

**Comment by iranzo** [comment]
I'm testing the excluded keywords "*.md" to get that filtered out, I'll report on the next checker run.

As a hint, would be good to have the allowed keywords listed in the readme in addition to the action.yml 

Thanks

**Comment by iranzo** [comment]
Closing the question, thanks!
